src/coinjoin/coinjoin-client.cpp

```diff
@@ -1042,6 +1042,12 @@ bool CCoinJoinClientSession::JoinExistingQueue(CAmount nBalanceNeedsAnonymized,
 
     auto mnList = deterministicMNManager->GetListAtChainTip();
 
+    int winners_to_skip{8};
+
+    if (Params().NetworkIDString() == CBaseChainParams::DEVNET || Params().NetworkIDString() == CBaseChainParams::REGTEST) {
+        winners_to_skip = 1;
```

Comment:
imo, this probably could / should be 0. Allow someone to mix if only 1 MN on net :shrug: 
```suggestion
        winners_to_skip = 0;
```

Reply:
I'm not sure how mixing on 1 MN could be useful (with no IS/CL) but ok :)